travis.yml

```diff
@@ -0,0 +1,26 @@
+language: node_js
+node_js:
+  - "8.9.4"
+
+branches:
+  only:
+    - master
+
+before_script:
+  - npm install -g @angular/cli
```

Comment:
don't install the cli, use the local version
also you'll need to do a local install `yarn install --frozen-lockfile` should be appropriate here

Reply:
@Toxicable No the ng build command will install dependencies

Reply:
No it wont. `ng build` cannot run if you don't have the cli installed.
And for CI you're not meant to install it globally since you're using a different version than whats in the project, this could very easily cause a failure